cypress/selectors/sign_in_page.js

```diff
@@ -29,17 +29,3 @@ export const sign_in_page = {
     log_out: "[data-test='sidenav-signout']",
     sidenav_username: "[data-test='sidenav-username']"
 }
-
```

Comment:
Rename "should show disabled by default sign in btn" test to "should show enabled by default sign in btn" and refactor it due to new name.